### DISCUSSION TITLE
Project and activity number variables

### DISCUSSION THREAD

**Discussion by rbuehler-teletrend** [kimai/kimai#4762] [General] [answered]
What would be the variables to access the project (and activity) numbers in invoice templates?
Something like `invoice['project.number']` seemed feasible, but didn't return a value.
Maybe this can be added to the invoice documentation?

**Comment by kevinpapst** [comment, marked as answer]
> Maybe this can be added to the invoice documentation?

Yeah, after I added it to the next release 😁 
And it will be `project.number` and `activity.number`
